vllm/worker/model_runner.py

```diff
@@ -655,6 +655,7 @@ def _prepare_model_input_tensors(
                 input_positions.append(0)
                 slot_mapping.append(_PAD_SLOT_ID)
                 seq_lens.append(1)
+                query_lens.append(1)
```

Comment:
Not used?

Reply:
This is used when calculating `query_start_loc`, which is the input for the flash attention backend when using the unified kernel.